packages/core/src/components/slides/slides.ios.vars.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../themes/ionic.globals.ios";
+
+/// @prop - Slides Bullet Background Color
+$slides-ios-bullet-background-color: $text-ios-color-step-200 !default;
```

Comment:
Can we fix indentation for all of these and the `md` file? Also the capitalization on the documentation is inconsistent.